lib/send-request.js

```diff
@@ -51,6 +51,9 @@ module.exports = function(location, options, callback) {
         break
     }
   }
+  else if (requestOptions.formData) {
```

Comment:
You should either add `var formData = requestOptions. formData || null` on line 9 to match the corresponding `requestParams` variable we use in this module, and then replace all other instances of `requestOptions.formData` with `formData` or remove `var requestParams = requestOptions.requestParams || null` from line 8 and replace all instances of `requestParams` with `requestOptions.requestParams`.